docs/modules/ROOT/pages/subscriptions/events/delete_relationship.adoc

```diff
@@ -0,0 +1,806 @@
+[[delete_relationship]]
+= Delete Relationship Subscriptions
+
+WARNING: Subscriptions are only available as a beta; its API may change in the future. It is not recommended to use subscriptions in production environments.
+
+Subscriptions to `DELETE_RELATIONSHIP` events will listen for relationships to a node of the specified type that have been deleted. 
+
+NOTE: This subscription operation is **only** available for types that define relationship fields.
+
+As relationship-specific information, the event will contain the relationship field name, as well as an object containing all relationship field names of the specified type. This object will be populated with properties according to the deleted relationship.
+
+NOTE: A new event will be triggered for each deleted relationship.
+
+This means that, if the type targeted for the subscriptions defines two or more relationships in the schema and one of each relationships are deleted following a mutation operation, there will be an equal number of events triggered as the number of deleted relationships. 
```

Comment:
I would apply the same grammar as I proposed on the `create-relationship` page to this sentence. 🙂